BankApplication/src/com/luxoft/bankapp/tests/TestBankService.java

```diff
@@ -0,0 +1,181 @@
+package com.luxoft.bankapp.tests;
+
+import static org.junit.Assert.*;
+
+import java.util.Arrays;
+import java.util.Set;
+
+import org.junit.Ignore;
+import org.junit.Test;
+
+
+import com.luxoft.bankapp.handling_exceptions.ClientExistsException;
+import com.luxoft.bankapp.handling_exceptions.NotEnoughFundsException;
+import com.luxoft.bankapp.handling_exceptions.OverDraftLimitExceededException;
+import com.luxoft.bankapp.model.Account;
+import com.luxoft.bankapp.model.Bank;
+import com.luxoft.bankapp.model.CheckingAccount;
+import com.luxoft.bankapp.model.Client;
+import com.luxoft.bankapp.model.Gender;
+import com.luxoft.bankapp.model.SavingAccount;
+import com.luxoft.bankapp.service.BankServiceImpl;
+
+import junit.framework.Assert;
+
+public class TestBankService {
+
+	private Bank bank=new Bank();
+	private BankServiceImpl bankService = new BankServiceImpl();
+	private Client client1 = new Client("Tom",1600,Gender.MALE, "Warsaw");
+	private Account savingAccount = new SavingAccount(300);
+	private Account checkingAccount = new CheckingAccount(300, client1);
+	private Account[] accounts= {savingAccount, checkingAccount};
+
+	//bankService.setActiveAccount(client1,checkingAccount);
+	
+	@Test
```

Comment:
I do not know how tests are working for you since JUnit included as test dependency in POM, but your tests are placed in runtime sources. So out of the box the JUnit is not plugged in.